src/components/data/PathListAutomation.js

```diff
@@ -213,229 +219,252 @@ export default function PathListAutomation(props) {
+                                                label={t("fileName")}
+                                                component={SaveAsField}
+                                            />
+                                        </Paper>
+                                    </Grid>
+                                </Grid>
+                            </DialogContent>
+                            <DialogActions>
+                                {createPathListError && (
+                                    <Grid item xs>
+                                        <ErrorTypographyWithDialog
+                                            baseId={baseId}
+                                            errorMessage={errorMsg}
+                                            errorObject={createPathListError}
+                                        />
+                                    </Grid>
+                                )}
+                                <Grid>
+                                    <Button
                                         id={buildId(
                                             baseId,
-                                            ids.PATH_LIST_AUTO_DEST_FIELD
+                                            ids.PATH_LIST_AUTO_CANCEL_BTN
                                         )}
-                                        acceptedType={ResourceTypes.FOLDER}
-                                        label={t("pathListDestLbl")}
-                                        component={InputSelector}
-                                        required={true}
-                                    />
-                                    <Field
+                                        onClick={onCancel}
+                                    >
+                                        {i18nCommon("cancel")}
+                                    </Button>
+                                </Grid>
+                                <Grid>
+                                    <Button
                                         id={buildId(
                                             baseId,
-
-                                            ids.PATH_LIST_AUTO_FILE_NAME_FIELD
+                                            ids.PATH_LIST_AUTO_DONE_BTN
                                         )}
-                                        name="fileName"
-                                        required={true}
-                                        label={t("fileName")}
-                                        component={SaveAsField}
-                                    />
-                                </Paper>
-                            </Grid>
-                        </Grid>
-                        <Grid
-                            container
-                            direction="row"
-                            justify="flex-end"
-                            alignItems="flex-end"
-                            spacing={1}
-                        >
```

Comment:
Looks like this `Grid` bit got missed when adding this to the `DialogActions` section